service.go

```diff
@@ -101,24 +102,24 @@ func (s *Service) listSubnets(ctx context.Context) ([]net.IPNet, error) {
 
 // CreateSubnet returns the next available subnet, of the configured size,
 // from the configured network.
-func (s *Service) CreateSubnet(ctx context.Context, mask net.IPMask, annotation string) (net.IPNet, error) {
-	s.logger.LogCtx(ctx, "level", "info", "message", "creating new subnet")
+func (s *Service) CreateSubnet(ctx context.Context, mask net.IPMask, annotation string, reserved []net.IPNet) (net.IPNet, error) {
```

Comment:
Just saying. The interface changes here, which is a breaking change in terms of versioning. Since this is all WIP it should work out but we should try to be more careful with these things.

Reply:
I was fully aware that interface changed here, but AFAIK this function should be only used from `kubernetesd` and there is already PR open for corresponding change (that was actually the driving force for this). Hence I didn't give this much more attention. Are you aware of other use cases of this?